Composer/package.json

```diff
@@ -40,7 +40,7 @@
     "start:client": "yarn workspace @bfc/client start",
     "start:server": "yarn workspace @bfc/server start",
     "start:server:dev": "yarn workspace @bfc/server start:dev",
-    "runtime": "cd ../BotProject/CSharp/ && dotnet build && dotnet run",
+    "runtime": "cd ../runtime/dotnet/azurewebapp && dotnet build && dotnet run",
```

Comment:
good start here. i imagine we'll expand these script targets to configure the hosting platform.. 

`runtime:azurewebapp`, `runtime:azurefunction`

or accept a cli parameter, ex. "language", "host"
"runtime": "cd ../runtime/${language}/${host} .."`

but even then i'm not sure we'll be happy with it given that dotnet, node, python etc have disparate api's and it will require reworking this quite a bit.

Reply:
Yes, this PR is just having what worked before, to work with the new structure. No new functionality. Having this be configurable is more of an R10 thing.

Reply:
For composer to support interchangeable runtimes, composer needs to make no assumptions about the runtimes so we agree on that one. There will be significant reworking proportionally, but in practice there are 4 classes in composer that do mos tof the runtime interaction: plugins (azure + local ), assetManager and BotDeployer. It's reasonable to rework those plus command line tools, considering that in exchange we gain interchangeable runtimes if we engineer it right.